README.md

```diff
@@ -68,6 +68,7 @@ disqusShortname = "XXX"
     shareTwitter = true
     shareFacebook = true
     shareGooglePlus = true
+    shareLinkedIn = true
```

Comment:
This has to default to `false` so that we do not enable this for existing users by default